apps/dcos_overlay/src/dcos_overlay_poller.erl

```diff
@@ -339,6 +493,34 @@ parse_vtep_mac(MAC) ->
         end,
         MACComponents).
 
+%%%===================================================================
+%%% Metrics functions
+%%%===================================================================
+
+-spec(init_metrics() -> ok).
+init_metrics() ->
+    prometheus_counter:declare([
+        {registry, overlay},
+        {name, mesos_state_polls_total},
+        {help, "Total number of times Mesos overlay state has beed polled."}]),
+    prometheus_counter:declare([
+        {registry, overlay},
+        {name, mesos_state_poll_errors_total},
+        {help, "Total number of errors while polling Mesos overlay state."}]),
+    prometheus_counter:declare([
+        {registry, overlay},
+        {name, networks_added_total},
+        {help, "Total number of overlay networks added."}]),
+    prometheus_counter:declare([
+        {registry, overlay},
+        {name, network_addition_errors_total},
+        {help, "Total number of errors while adding overlay networks."}]),
```

Comment:
IIRC poller configures only `vtep` and doesn't add new overlay networks, so maybe `vtep_configuration_errors_total`?